src/db/models/job.ts

```diff
@@ -38,13 +40,28 @@ export const PDF_FILE_RECORD = t.type({
   projectId: PROJECT_ID,
 });
 
+export const EXCEL_FILE_RECORD = t.type({
+  name: t.string,
+  planId: PLAN_ID,
+});
+
 export const JOB_METADATA_PROJECT_PDF = t.partial({
   startedBy: PARTICIPANT_ID,
   processed: t.number,
   total: t.number,
   files: t.array(PDF_FILE_RECORD),
 });
 
+export const JOB_METADATA_PROJECT_EXCEL = t.partial({
+  startedBy: PARTICIPANT_ID,
+  files: EXCEL_FILE_RECORD,
```

Comment:
If it's no longer an array, this should be `file`, not `files`
```suggestion
  file: EXCEL_FILE_RECORD,
```